### PR TITLE
MAUI-884496 Prepared UG document for Axis element position and Zoom Pan event in .NET MAUI Charts

### PR DESCRIPTION
**Pull request by snehakPV**
**Description**

- Prepared UG document for Axis element position
- Prepared UG document for Zoom pan event

**Reference**

Xamarin:

- Axis labels and ticks - https://help.syncfusion.com/xamarin/charts/axis#label-and-tick-positioning
- Zoom pan event - https://help.syncfusion.com/xamarin/charts/zoompan#events

WPF: 

- Axis label - https://help.syncfusion.com/wpf/charts/axis#axis-labels
- Tick lines - https://help.syncfusion.com/wpf/charts/axis#tick-lines

### CODE REVIEW

File: MAUI/Cartesian-Charts/Axis/Ticklines.md [hunk 1]
```diff
@@ -50,6 +50,42 @@ chart.YAxes.Add(secondaryAxis);
 
 {% endtabs %}
 
+## Positioning the ticklines
+
+The positioning of tick lines inside or outside the chart area can be controlled using the [TickPosition]() property. By default, the tick lines are positioned `AxisElementPosition.Outside` the chart area.
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/Axis/Ticklines.md:55]: Improve the default value like follows -https://help.syncfusion.com/maui/cartesian-charts/axis/axislabels#edge-labels-drawing-mode

File: MAUI/Cartesian-Charts/Axis/Axislabels.md [hunk 2]
```diff
@@ -11,6 +11,42 @@ documentation: ug
 
 Axis labels are used to show the units or measures or category value of axis to visualize the data user friendly. It will be generated based on the range and the values binded to [XBindingPath](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartSeries.html#Syncfusion_Maui_Charts_ChartSeries_XBindingPath) or [YBindingPath](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.XYDataSeries.html#Syncfusion_Maui_Charts_XYDataSeries_YBindingPath) properties of series.
 
+## Positioning the labels
+
+The [LabelsPosition]() property is used to position the axis labels inside or outside the chart area. By default, [LabelsPosition]() is positioned `AxisElementPosition.Outside` the chart area.
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/Axis/Axislabels.md:16]: Improve the default value like follows -https://help.syncfusion.com/maui/cartesian-charts/axis/axislabels#edge-labels-drawing-mode

File: MAUI/Polar-Charts/Axis/Axislabels.md [hunk 3]
```diff
@@ -11,6 +11,44 @@ documentation: ug
 
 Axis labels are used to display the units, measures, or category values of an axis in a user-friendly way. They are generated based on the range and the values bound to the [XBindingPath](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartSeries.html#Syncfusion_Maui_Charts_ChartSeries_XBindingPath) or [YBindingPath](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.XYDataSeries.html#Syncfusion_Maui_Charts_XYDataSeries_YBindingPath) properties of the series.
 
+## Positioning the labels
+
+The [LabelsPosition]() property is used to position the axis labels inside or outside the chart area. By default, [LabelsPosition]() is positioned `AxisElementPosition.Outside` the chart area.
+
```
Comment (SaiyathAliFathima) [inline MAUI/Polar-Charts/Axis/Axislabels.md:17]: Improve the default value like follows -https://help.syncfusion.com/maui/cartesian-charts/axis/axislabels#edge-labels-drawing-mode

File: MAUI/Polar-Charts/Axis/Ticklines.md [hunk 4]
```diff
@@ -49,6 +49,43 @@ chart.SecondaryAxis = secondaryAxis;
 
 {% endtabs %}
 
+## Positioning the ticklines
+
+The positioning of tick lines inside or outside the chart area can be controlled using the [TickPosition]() property. By default, the tick lines are positioned `AxisElementPosition.Outside` the chart area.
```
Comment (SaiyathAliFathima) [inline MAUI/Polar-Charts/Axis/Ticklines.md:54]: Improve the default value like follows -https://help.syncfusion.com/maui/cartesian-charts/axis/axislabels#edge-labels-drawing-mode

File: MAUI/Cartesian-Charts/Axis/Axislabels.md [hunk 5]
```diff
@@ -11,6 +11,42 @@ documentation: ug
...
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfCartesianChart>
+    . . .
+    <chart:SfCartesianChart.YAxes>
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/Axis/Axislabels.md:24]: Add x axis code as we discussed for cartesian chart

File: MAUI/Cartesian-Charts/Axis/Ticklines.md [hunk 6]
```diff
@@ -50,6 +50,42 @@ chart.YAxes.Add(secondaryAxis);
...
+
+The positioning of tick lines inside or outside the chart area can be controlled using the [TickPosition]() property. By default, the tick lines are positioned `AxisElementPosition.Outside` the chart area. 
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfCartesianChart>
+    . . .  
+    <chart:SfCartesianChart.YAxes>
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/Axis/Ticklines.md:63]: Add x axis code as we discussed for cartesian chart

File: MAUI/Polar-Charts/Axis/Axislabels.md [hunk 7]
```diff
@@ -11,6 +11,44 @@ documentation: ug
...
+N> Polar chart supports LabelsPosition for secondary axis only.
```
Comment (SaiyathAliFathima) [inline MAUI/Polar-Charts/Axis/Axislabels.md:18]: This is only applicable to the secondary axis of the polar chart.

File: MAUI/Polar-Charts/Axis/Ticklines.md [hunk 8]
```diff
@@ -49,6 +49,43 @@ chart.SecondaryAxis = secondaryAxis;
...
+
+N> Polar chart supports TickPosition for secondary axis only.
+
```
Comment (SaiyathAliFathima) [inline MAUI/Polar-Charts/Axis/Ticklines.md:57]: This is only applicable to the secondary axis of the polar chart.

Comment (snehakPV) [reply MAUI/Polar-Charts/Axis/Ticklines.md:57]: Updated the content

File: MAUI/Polar-Charts/Axis/Axislabels.md (hunk 7 above)
Comment (snehakPV) [reply MAUI/Polar-Charts/Axis/Axislabels.md]: Updated the content

File: MAUI/Cartesian-Charts/Axis/Ticklines.md (hunk 6 above)
Comment (snehakPV) [reply MAUI/Cartesian-Charts/Axis/Ticklines.md:68]: Updated the code snippet

File: MAUI/Cartesian-Charts/Axis/Axislabels.md (hunk 5 above)
Comment (snehakPV) [reply MAUI/Cartesian-Charts/Axis/Axislabels.md:29]: Updated the code snippet

File: MAUI/Polar-Charts/Axis/Ticklines.md (hunk 4 above)
Comment (snehakPV) [reply MAUI/Polar-Charts/Axis/Ticklines.md]: Updated the content as present in the link

File: MAUI/Polar-Charts/Axis/Axislabels.md (hunk 3 above)
Comment (snehakPV) [reply MAUI/Polar-Charts/Axis/Axislabels.md:17]: Updated the content as present in the link

File: MAUI/Cartesian-Charts/Axis/Axislabels.md (hunk 2 above)
Comment (snehakPV) [reply MAUI/Cartesian-Charts/Axis/Axislabels.md]: Updated the content as present in the link

File: MAUI/Cartesian-Charts/Axis/Ticklines.md (hunk 1 above)
Comment (snehakPV) [reply MAUI/Cartesian-Charts/Axis/Ticklines.md]: Updated the content as present in the link

**Review by SaiyathAliFathima — APPROVED** [review]

**Pull request closed without merging**